sideci_demo/phpmd_demo.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use Github\Client;
+
+class Repository
+{
+    private $owner;
+    private $name;
+    private $public;
+    private $branches;
+
+    const PublicVisibility = 'public';
+
+    /**
+     * @param string $owner
+     * @param string $name
+     * @param string $visibility
+     */
+    public function Repository(string $owner, string $name, string $visibility)
```

Comment:
__[PHP_CodeSniffer]__

Method name "Repository::Repository" is not in camel caps format


<!-- Comment by [Sider](https://sider.review) -->